rubix-spi/src/main/java/com/qubole/rubix/spi/fop/ObjectPoolPartition.java

```diff
@@ -56,11 +56,14 @@ public ObjectPoolPartition(ObjectPool<T> pool, PoolConfig config,
     this.log = new CustomLogger(name, host);
     this.takeSemaphore = new Semaphore(config.getMaxSize(), true);
     for (int i = 0; i < config.getMinSize(); i++) {
-      T object = objectFactory.create(host, socketTimeout, connectTimeout);
-      if (object != null) {
+      try {
+        T object = objectFactory.create(host, socketTimeout, connectTimeout);
         objectQueue.add(new Poolable<>(object, pool, host));
         totalCount++;
       }
+      catch (Exception e) {
```

Comment:
move full `for` loop inside try-catch. Otherwise we will generate one warn log for each trial

Reply:
this is log is actually not needed, factories are logging cases of exception

Reply:
removed